contrib/test/run_test_vectors.sh

```diff
@@ -42,18 +48,20 @@ else
   exit 0
 fi
 
-find dump/test-vectors/syscall/fixtures -type f -name '*.fix' -exec ./$OBJDIR/unit-test/test_exec_sol_compat {} + > $LOG_PATH/test_vectors_exec 2>&1
-test_res=$?
-failed=`grep -wR FAIL $LOG_PATH | wc -l`
-echo "Total failed: $failed"
+TESTS=dump/test-vectors/syscall/fixtures
+LOG=$LOG_PATH/test_exec_syscalls
+find $TESTS -type f -name '*.fix' -exec ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG {} +
+if [ "$?" != "0" ]
```

Comment:
test_exec_sol_compat already checks if any test fails, and together with the set -e it's safe to just call it and let it fail CI.

I added the if just as an example and to terminate early (but if someone copy&paste and forgets the if, we're good).
There's no need to tail the error logs, because test_exec_sol_compat outputs warnings.